chapters/scoping.tex

```diff
@@ -79,7 +79,7 @@ \subsection{Simple Name Lookup}\label{simple-name-lookup}
   Among declared named elements (\lstinline!class-definition! and \lstinline!component-declaration!) of the class (including elements inherited from base classes).
 \item
   Among the import names of qualified \lstinline!import!-clauses in the \emph{instance} scope.
-  The import name of \lstinline!import A.B.C!; is \lstinline!C! and the import name of \lstinline!import D=A.B.C;! is \lstinline!D!.
+  The import name of \lstinline!import A.B.C!; is \lstinline!C! and the import name of \lstinline!import D = A.B.C;! is \lstinline!D!.
```

Comment:
I wouldn't call it "binary operator" here. However, even if I don't see the need for the spaces I can accept them.

Reply:
OK.  Resolving.